docs/models/YOLOS/input_variations.md

```diff
@@ -14,7 +14,7 @@
 | 10 | aten.convolution.default       |                  1 |           0 |
 | 11 | aten.div.Tensor                |                  1 |           0 |
 | 12 | aten.expand.default            |                  5 |           0 |
-| 13 | aten.floor.default             |                  2 |           2 |
+| 13 | aten.floor.default             |                  2 |           0 |
```

Comment:
These results are generated from a Grayskull machine, and this PR https://github.com/tenstorrent/pytorch2.0_ttnn/pull/174 added checks to skip incompatible ops from lowering, such as `ttnn.floor`. So previously, the op was lowered successfully (even if it doesn't run). But now it will not lower if the device is Grayskull.